pkg/templates/node_1.10.go

```diff
@@ -212,6 +212,20 @@ systemd:
         Type=oneshot
         [Install]
         WantedBy=multi-user.target
+    - name: fix-cert-rotation.service
+      command: start
+      enable: true
+      contents: |
+        [Unit]
+        Description=Fix kubelet certificate rotation
+        After=kubelet.service
+        [Service]
+        Type=simple
+        Restart=on-failure
+        RestartSec=5
```

Comment:
Maybe we should relax this to 60 or even 300 seconds? There is not need to retry that often, we have no hurry in fixing it, no? If this fails constantly, retrying every 5 seconds seems excessive

Reply:
You are right, there is no hurry. I changed this to 300s.